base-system/usrroot/usr/sbin/hos-wallpaper

```diff
@@ -66,7 +66,7 @@ prepare_wallpaper(){
 	SECTION="$1"
 	CURRENT_MODE="$2"
 	WALLPAPER_URL="$(hos-dvar --variable-name Wallpaper --section "$SECTION")"
-	WALLPAPER_HASH_URL="${WALLPAPER_URL}.sha256"
+	WALLPAPER_HASH="$(hos-dvar --variable-name WallHash --section "$SECTION")"
```

Comment:
With the new PR #67 I definitely preffer the explicit name now. 'WallpaperHash' or 'WalpaperSha256' sounds good.

Reply:
I think WallpaperSha256 is explicit enough